tests/e2e/workloads/pvc_snapshot_and_clone/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import random
+import os
+import urllib.request
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import bugzilla
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources import pod, pvc
+from ocs_ci.helpers import helpers
+from tests.conftest import snapshot_factory
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    tier2,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+
+log = logging.getLogger(__name__)
+
+
+@tier2
```

Comment:
this doesnt seem to be tier2, please move it to tier1

Reply:
sure